ApplyToBecomeInternal/ApplyToBecomeInternal/TagHelpers/DateInputTagHelper.cs

```diff
@@ -22,10 +22,17 @@ public DateInputTagHelper(IHtmlHelper htmlHelper, ErrorService errorService) : b
 		}
 
 		protected override async Task<IHtmlContent> RenderContentAsync()
+		{
+			DateInputViewModel model = ValidateRequest();
+
+			return await _htmlHelper.PartialAsync("_DateInput", model);
+		}
+
+		private DateInputViewModel ValidateRequest()
 		{
 			if (For.ModelExplorer.ModelType != typeof(DateTime?))
 			{
-				throw new ArgumentException();
+				throw new ArgumentException("ModelType is not a DateTime?");

```

Comment:
Is this saying that ModelType is not a nullable datetime or is it an unknown, either way perhaps being explicit will help? I'm just desperately looking for something to comment on so feel free to ignore 😃

Reply:
I'm not sure I follow, would you mind rephrasing. Sorry Monday morning.

Reply:
Have re-worded the exception message.